### PR TITLE
Task/v 9

### PR DESCRIPTION
**Pull request by XavierVal**
Test updates for v9, 
added versions checks for STH and ORC

### CODE REVIEW

**Comment by dmoranj** [comment]
LGTM

File: tests/properties.json [hunk 1]
```diff
@@ -52,6 +53,13 @@
       "protocol": "http",
       "version": ""
     },
+    "STH": {
+      "basiccheck": "/",
+      "heartbeat": "/",
+      "port": "8666",
```
Comment (AlvaroVega) [inline tests/properties.json]: with pep? I guess we still are without peps

**Comment by AlvaroVega** [comment]
LGTM

File: tests/properties.json (hunk 1 above)
Comment (XavierVal) [inline tests/properties.json]: Right now without PEP: 
STH_PORT: The port where the STH server will be listening. Optional. Default value: 8666.
Via: [IoT-STH](https://github.com/gtorodelvalle/IoT-STH)

**Pull request merged**